PayButton/paybutton-server
# [Issue #177] feat: add logic to sync transactions after login

**Description:**
Part of #177 
Added signin / signup callbacks, as described in [the supertokens docs](https://supertokens.com/docs/thirdpartyemailpassword/common-customizations/handling-signinup-success).
Post login now triggers the syncing of the transactions related to each one of the logged in user's addresses.

**Test plan:**
With the BCH / XEC nodes URLS defined in the `.env` file such as:
```
GRPC_BCH_NODE_URL=bchd.fountainhead.cash:443    
GRPC_XEC_NODE_URL=grpc.fabien.cash:8335         
```
...do:
1. `make reset-dev`
2. Log in into the default dev user on http://localhost:3000
3. Go to buttons, select any of them. All their transactions should appear on the button detail page (the syncing of the transactions may take some seconds, from the moment of the login, to happen).
